lucet-runtime/src/lib.rs

```diff
@@ -369,6 +368,10 @@ pub mod vmctx {
     //! associated with a running instance. This should never occur if run in guest code on the
     //! pointer argument inserted by the compiler.
     pub use lucet_runtime_internals::vmctx::{lucet_vmctx, Vmctx};
+
+    // must be exported for `lucet_hostcall`, but we don't want to advertise it
+    #[doc(hidden)]
+    pub use lucet_runtime_internals::vmctx::VmctxInternal;
```

Comment:
as w/ the deprecated attributes, this is nice attention to detail :+1: great work